modules/drivers/googleanalytics/src/metabase/driver/googleanalytics/query_processor.clj

```diff
@@ -361,14 +384,16 @@
   (when order-by
     {:sort (str/join
             ","
-            (for [[direction field] order-by]
-              (str (case direction
-                     :asc  ""
-                     :desc "-")
-                   (mbql.u/match-one field
-                     [:datetime-field _ unit] (unit->ga-dimension unit)
-                     [:aggregation index]     (mbql.u/aggregation-at-index query index)
-                     [& _]                    (->rvalue &match)))))}))
+            (remove
+             str/blank?
+             (for [[direction field] order-by]
+               (str (case direction
+                      :asc  ""
+                      :desc "-")
+                    (mbql.u/match-one field
+                      [:datetime-field _ unit] (unit->ga-dimension unit)
+                      [:aggregation index]     (mbql.u/aggregation-at-index query index)
+                      [& _]                    (->rvalue &match))))))}))
```

Comment:
for my tasted I'd prefer a

```
(->> order-by
  (map ...)
  (remove str/blank?)
  (str/join ","))
  ```
  since I'm not a big fan of deeply-nested `for`s, but it doesn't really matter

Reply:
yeah, that would be nicer. I'll change it